sycl/test/filter_selector/select.cpp

```diff
@@ -0,0 +1,133 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: %t1.out
+
+//==------------------- select.cpp - string_selector test ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+using namespace cl::sycl::ONEAPI;
+
+int main() {
+  std::vector<device> CPUs;
+  std::vector<device> GPUs;
```

Comment:
we can also process FPGA emulator devices in the same way

Reply:
Ok - those should identify as accelerator, right?

Reply:
yes